contrib/browndog/PEcAn#DALEC_convert.R

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/Rscript
-#PEcAn
-#data
-#pecan.zip
-#dalec
+# PEcAn
```

Comment:
I'd hope not, if it does we should discuss this with the polyglot people and have them fix it :)